d/drupal-ultimate_cron/drupal-ultimate_cron_ubi_8.5.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash -e
+
+# ----------------------------------------------------------------------------
+# Package          : drupal-ultimate_cron
+# Version          : 7.x-2.x
+# Source repo      : https://git.drupalcode.org/project/ultimate_cron
+# Tested on        : UBI 8.5
+# Language         : PHP
+# Travis-Check     : True
+# Script License   : Apache License, Version 2 or later
+# Maintainer       : Haritha Patchari <haritha.patchari@ibm.com>
+#
+# Disclaimer       : This script has been tested in root mode on given
+# ==========         platform using the mentioned version of the package.
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#   
+# -----------------------------------------------------------------------------
+
+# Variables
+PACKAGE_NAME=ultimate_cron
+CORE_PACKAGE_NAME=drupal
+PACKAGE_URL=https://git.drupalcode.org/project/ultimate_cron
+CORE_PACKAGE_URL=https://github.com/drupal/drupal
+#PACKAGE_VERSION is configurable can be passed as an argument.
+PACKAGE_VERSION=${1:-7.x-2.0}
```

Comment:
The version here is not matching with the version at line no 5.

Reply:
@kumarvikas2020 Made changes to match version